src/bin/signal_handler.rs

```diff
@@ -14,28 +14,28 @@
 
 #[cfg(unix)]
 mod imp {
-    use std::time::Duration;
     use std::sync::Arc;
-    use std::sync::atomic::AtomicBool;
 
     use rocksdb::DB;
+    use libc;
 
     use tikv::server::Msg;
     use tikv::util::transport::SendCh;
     use prometheus::{self, Encoder, TextEncoder};
 
+    // Real-time signals
+    const TOGGLE_PROF_SIG: libc::c_int = 40;
```

Comment:
Yes, we thought about it. But rust grpc is not stable enough, so signal is still used here.